src/i18n/locale/en.json

```diff
@@ -8,19 +8,19 @@
     },
     "error": {
       "label": {
-        "empty": "Category is Empty",
+        "empty": "Empty Category",
```

Comment:
```suggestion
        "empty": "Empty category",
```
?

Reply:
don't change this, this is a message shown to the user when a category is empty.